src/components/SignIn.js

```diff
@@ -30,17 +35,85 @@ function Copyright(props) {
 }
 
 export default function SignIn() {
-  const handleSubmit = (event) => {
+  const [formData, setFormData] = useState({
+    email: '',
+    password: '',
+  });
+
+  //update the input states when their value is changed
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData((prevData) => ({ ...prevData, [name]: value }));
+  };
+
+  const navigate = useNavigate();
+
+  // snackbar
+  const [open, setOpen] = React.useState(true);
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+  };
+  const action = (
+    <React.Fragment>
+      <IconButton
+        size="small"
+        aria-label="close"
+        color="inherit"
+        onClick={handleClose}
+      >
+        <CloseIcon fontSize="small" />
+      </IconButton>
+    </React.Fragment>
+  );
+  // end of snackbar
+
+  // user authentication
+  const [loginError, setLoginError] = React.useState(false);
```

Comment:
`loginError` is no longer used so we can remove this and all the places we're calling `setLoginError`

Reply:
@JafariM I think this still applies; maybe you fixed it before but it looks like it's still there, maybe it got undone with the last merge or something?

Reply:
> 

Yes it was undone, I just delete them agian